Modules/Server/Sources/PocketCastsServer/Private/Protobuffer/api.pb.swift

```diff
@@ -200,6 +200,8 @@ struct Api_UserLoginResponse {
 
   var uuid: String = String()
 
+  var email: String = String()
```

Comment:
I assume this is available only on Staging, right?

In that case, we probably wouldn't want to merge this change to `trunk`.

Reply:
This field is only available on staging, but I think the decision to merge or not might be based on the concern you're seeing. 
I see two possible concerns:

1) Will this impact login if the value isn't available?
- No, the email will default to nil and be ignored.

2) What happens if we regenerate the protobuff from `trunk`?
- The developer would get an error, and we can regenerate from Staging, which will include all of prod plus what's in development. I can see this as a reason not to merge, but I think its a small one.

Reply:
If everything works as it is I think we're good. :)